### ISSUE TITLE
[BUG] [0.58.0] failed tests

### ISSUE THREAD

**Issue by remicollet** [DataDog/dd-trace-php#1214]
```
=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
HTTP headers are sent to the Agent from the background sender [tests/ext/background-sender/agent_headers.phpt]
The container ID is sent via HTTP headers to the Agent [tests/ext/background-sender/agent_headers_container_id.phpt]
An empty container ID is not sent via HTTP headers to the Agent [tests/ext/background-sender/agent_headers_container_id_empty.phpt]
The Fargate 1.4+ container ID is sent via HTTP headers to the Agent [tests/ext/background-sender/agent_headers_container_id_fargate.phpt]
HTTP Agent headers are ignored from userland [tests/ext/background-sender/agent_headers_ignore_userland.phpt]
=====================================================================
```

All failing with `getaddrinfo failed` (http://request-replayer)

It will be nice to have such tests, relying on dev env skipped.

**Comment by SammyK** [comment]
Thanks for the report @remicollet. This should be fixed in the next release via #1215.

**Comment by labbati** [comment]
[0.59.0(https://github.com/DataDog/dd-trace-php/releases/tag/0.59.0) has been released and it solves this issue. In case of any problem please comment and we will reopen.
